### PR TITLE
Implement backup details page

### LINKED ISSUE #492: Implement backup details and restore page

### PR DESCRIPTION
**Pull request by briansteffens**
Closes #492

screen shot 2016-12-15 at 10 45 01 am

### CODE REVIEW

File: src/linodes/linode/backups/layouts/BackupPage.js [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+              <label>
+                <input
+                  id="destroy-all"
+                  type="checkbox"
+                  value={overwrite}
+                  name="overwrite"
+                  onChange={() => this.setState({ overwrite: !overwrite })}
+                />
+                Destroy all current disks and backups
+              </label>
```
Comment (eatonphil) [inline src/linodes/linode/backups/layouts/BackupPage.js]: Check for usage of the `checkbox` class to get better styles here.

File: src/linodes/linode/backups/layouts/BackupPage.js [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+      </div>
+    ), (
+      <div className="form-group row" key="button">
+        <div className="col-sm-3 label-col"></div>
+        <div className="col-sm-8 content-col right">
+          <button
+            type="button"
+            className="btn btn-default"
+            name="takeSnapshot"
+            onClick={this.takeSnapshot}
```
Comment (eatonphil) [inline src/linodes/linode/backups/layouts/BackupPage.js]: There is a modal that pops up if a backup would _require_ you to overwrite your current disks. (In addition to that overwrite field). But that can be handled in another task/pr.

Comment (eatonphil) [inline src/linodes/linode/backups/layouts/BackupPage.js]: #800

**Comment by eatonphil** [comment]
Talked about it in meetspace, but commenting so I don't forget and just merge this: page should probably change (dunno where to).

**Comment by briansteffens** [comment]
A restore can take a long time, maybe it should show a progress indicator or something

**Pull request merged**